pkg/splunk/enterprise/names.go

```diff
@@ -124,6 +124,37 @@ const (
 	livenessProbeDefaultDelaySec = 300
 	livenessProbeTimeoutSec      = 30
 	livenessProbePeriodSec       = 30
+
+	// Number of ClusterMasterReplicas
+	numberOfClusterMasterReplicas = 1
+
+	// Number of Deployer replicas
+	numberOfDeployerReplicas = 1
+
+	// Number of Licensemaster replicas
+	numberOfLicenseMasterReplicas = 1
+
+	// Telemetry app.conf string
+	telAppConfString = `[install]
+is_configured = 1
+	
+[ui]
+is_visible = 0
+label = Splunk Operator for K8s
```

Comment:
Do you think, we can also inject the splunk personality, that way, we can find how many instances of a given kind?

Reply:
The contents of the app.conf here or any apps in general are not accessible via telemetry. The role of the Splunk instance needs to be determined via other collected fields. One idea is to name the app according to the role?

Reply:
Changed the app name to include the kind of deployment, thanks for the suggestion @sgontla